pkg/csi/service/node.go

```diff
@@ -1365,7 +1365,7 @@ func getDevFromMount(target string) (*Device, error) {
 		if m.Path == target {
 			// something is mounted to target, get underlying disk
 			d := m.Device
-			if m.Device == "udev" {
+			if m.Device == "udev" || m.Device == "devtmpfs" {
```

Comment:
Could you also add a sample mount output you are receiving for the raw block volume after line 1348 in this file? This will help us debug issues in future.

Reply:
Sure, I've added an example, taken from a RAW block PVC deployment on OpenShift 4.5.7. I hope this is ok!